assets/css/forms.css

```diff
@@ -143,120 +164,79 @@ button:hover {
 }
 
 .header__description {
-  display: grid;
-  justify-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
 .description__title {
-  font-size: 20px;
+  text-align: center;
+  font-size: 28px;
 }
 
 .description__paragraph {
   font-size: 15px;
   padding: 0 3rem;
-  text-align: justify;
+  text-align: center;
 }
 
-.contact__form-wrapper {
-  margin: 25px 0;
-}
 
-.contact__header,
+/*** RIGHT COLUMN ***/
 .contact__form {
   display: flex;
   flex-direction: column;
-}
-
-.contact__header {
-  align-items: center;
+  justify-content: center;
+  align-content: center;
 }
 
 .form__label {
-  margin: 5px 0;
+  margin: 15px 0 5px;
   color: var(--purple);
   font-weight: 800;
-  font-size: 15px;
+  font-size: 20px;
+  max-width: 200px;
 }
 
 .form__input {
   margin-bottom: 8px;
   height: 35px;
-}
 
-.form__text {
-  margin-bottom: 8px;
-  height: 90px;
 }
 
 .form__button {
   text-transform: uppercase;
   font-size: 12px;
   line-height: 20px;
-  padding: 10px 30px;
+  padding: 10px 60px;
 }
 
-@media (max-width: 768px) {
-  .contact__main {
-    display: grid;
-    grid-template-columns: none;
-    grid-template-rows: repeat(2, 1fr);
-    margin: 0;
-  }
-
-  .description__paragraph {
-    padding: 0 7rem;
-  }
-
-  .contact__header-wrapper {
-    margin-top: 4rem;
-  }
-
-  .contact__form-wrapper {
-    margin: 0;
-  }
-}
+@media (max-width: 800px) {
 
-@media (max-width: 426px) {
   .contact__main {
-    margin: 0;
+    flex-direction: column;
+    margin-top: 80px;
+    margin-bottom: 80px;
+    width: 100%;
   }
 
-  .header__image {
-    height: 150px;
-    width: 150px;
+  .contact__header-wrapper,
+  .contact__form-wrapper {
+    width: 80%;
   }
 
   .description__title {
-    font-size: 15px;
+    font-size: 20px;
   }
-
   .description__paragraph {
-    font-size: 13px;
-    padding: 0px 3rem;
-  }
-
-  .contact__form-wrapper {
-    margin: 0;
-  }
-
-  .form__label,
-  .form__input,
-  .form__text {
-    width: 260px;
-  }
-  .form__button {
-    padding: 8px 24px;
+    padding: 0;
   }
-}
 
-@media (max-width: 320px) {
-  .description__paragraph {
-    padding: 0px 2rem;
+  .contact__form {
+    margin-top: 40px;
+    width: 90%;
   }
 
-  .contact__form-wrapper {
-    margin: 0;
-  }
 }
```

Comment:
- we removed multiple media queries and replaced it with just one
- we used percentage values to make sure our content automatically gets sized
- we reduced font sizing for titles and paragraphs